climakitaegui/core/data_interface.py

```diff
@@ -128,52 +128,55 @@ def _map_view(selections, stations_gdf):
     ax.set_extent(extent, crs=xy)
 
     # Set size of markers for stations depending on map boundaries
-    if extent == ca_extent:
-        scatter_size = 4.5
-    elif extent == us_extent:
-        scatter_size = 2.5
-    elif extent == na_extent:
-        scatter_size = 1.5
-
-    if selections.resolution == "45 km":
-        _add_res_to_ax(
-            poly=_wrf_bb["45 km"],
-            ax=ax,
-            color="green",
-            rotation=28,
-            xy=(-154, 33.8),
-            label="45 km",
-        )
-    elif selections.resolution == "9 km":
-        _add_res_to_ax(
-            poly=_wrf_bb["9 km"],
-            ax=ax,
-            color="red",
-            rotation=32,
-            xy=(-134, 42),
-            label="9 km",
-        )
-    elif selections.resolution == "3 km":
-        _add_res_to_ax(
-            poly=_wrf_bb["3 km"],
-            ax=ax,
-            color="darkorange",
-            rotation=32,
-            xy=(-127, 40),
-            label="3 km",
-        )
+    match extent:
+        case extent if extent == ca_extent:
```

Comment:
formatting isn't quite right here, it should be
```python
match extent:
    case ca_extent:
        # logic
    case us_extent:
        # logic
    case na_extent:
        # logic
    case _:
        # none of the above, do something drastic
```

Reply:
Ahh it actually throws an error if you have a value as a case like that...strange

Reply:
Oh, it's probably comparing classes instead of comparing values. I'll have a quick poke around to see if there's a better way to do this.

Reply:
This is the error I get in vscode when I try to use `case ca_extent`:

`Irrefutable pattern is allowed only for the last case statement`

Reply:
Looks like the correct implementation is either:
```python
match extent:
    case _ if extent == ca_extent:
        # logic
    case  _ if extent == us_extent:
        # logic
    case  _ if extent == na_extent:
        # logic
    case _:
        # none of the above, do something drastic
```
which I'm fine with. OR you could replace this with the actual values of the variables. But I'm happy to go with the option above.

Reply:
Yep, silly python...

Fixed in [72e4133](https://github.com/cal-adapt/climakitaegui/pull/29/commits/72e41330faaedf35f34c4dfc62833654808224e4)